mmv1/templates/terraform/pre_create/network_security_security_profile_group_operation.go.erb

```diff
@@ -0,0 +1 @@
+project = ""
```

Comment:
why are these custom code files necessary?

Reply:
This resource behaves like AuthorizationPolicy (always in networksecurity).
When I tried without the project it threw an error saying something like operation was missing a field.
I then did the same thing that was implemented in AuthorizationPolicy and things started to magically worked :)
While it worked empirically I also wanted to ask the exact meaning of this.

FYI in the yaml you should find:
```
custom_code:
  !ruby/object:Provider::Terraform::CustomCode # Calling custom operation that are different from other network security resources.
  pre_create: templates/terraform/pre_create/network_security_security_profile_group_operation.go.erb
  pre_update: templates/terraform/pre_update/network_security_security_profile_group_operation.go.erb
  pre_delete: templates/terraform/pre_delete/network_security_security_profile_group_operation.go.erb
```

Reply:
As per my comment [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/9800#discussion_r1451424324) this custom_code thing is not needed at all so I'll move forward removing it. `include_project: true` is enough to make things work.